docker-multinode/common.sh

```diff
@@ -274,9 +269,9 @@ kube::multinode::restart_docker(){
       else
         DOCKER_CONF="/etc/default/docker"
         
-        kube::helpers::file_replace_line ${DOCKER_CONF} `# Replace content in this file` \
-          "--bip" `# Find a line with this content...` \
-          "OPTIONS=\"\$OPTIONS --mtu=${FLANNEL_MTU} --bip=${FLANNEL_SUBNET}\"" `# ...and replace the found line with this line`
+        kube::helpers::file_replace_line ${DOCKER_CONF} \ # Replace content in this file
```

Comment:
The changes here don't work. On Ubuntu 14.04, at least, I get these errors:

./common.sh: line 273: --bip: command not found
./common.sh: line 274: DOCKER_OPTS="$DOCKER_OPTS --mtu=1472 --bip=10.1.79.1/24": No such file or directory

The trailing comment doesn't work, as the backslash needs to be the last character on the line. Maybe put all the comments above the function call, something like 

# Replace lines containing "--bip" in the ${DOCKER_CONF} file with the last line.